airflow/example_dags/example_params_trigger_ui.py

```diff
@@ -33,7 +33,7 @@
 
 with DAG(
     dag_id=Path(__file__).stem,
-    description=__doc__[0 : __doc__.find(".")],
+    description=__doc__.partition(".")[0],
```

Comment:
Not totally related but gosh I’m not sure we should encourage this pattern.

Reply:
It does have an appeal, especially if you consider the D400 rule of pydoc style.